build.gradle

```diff
@@ -45,6 +45,11 @@ shadowJar {
     }
 }
 
+jar {
+    dependsOn("shadowJar")
+    enabled = false
+}
+
```

Comment:
https://github.com/Iridium-Development/IridiumSkyblock/blob/accb95c813029bbca9ae2f4b6eddc06b9aec562e/build.gradle.kts is written in Kotlin and this `jar` thing is under `tasks` with a bunch of other stuff. Look into it a bit more.

Try adding all of this: https://github.com/Iridium-Development/IridiumSkyblock/blob/accb95c813029bbca9ae2f4b6eddc06b9aec562e/build.gradle.kts#L60-L110
*But remove obvious stuff like https://github.com/Iridium-Development/IridiumSkyblock/blob/accb95c813029bbca9ae2f4b6eddc06b9aec562e/build.gradle.kts#L73-L76 and change stuff like https://github.com/Iridium-Development/IridiumSkyblock/blob/accb95c813029bbca9ae2f4b6eddc06b9aec562e/build.gradle.kts#L68*